K2Bridge.Tests.End2End/KustoClientTests.cs

```diff
@@ -90,80 +90,83 @@ public async Task MSearch_All_ReturnsAllHitsAsExpected()
         public async Task FieldCaps_WithAllTypes_ReturnsTypesConvertedToESTypes()
         {
             var fieldCaps = await K2Client().FieldCaps(typesIndexFullName);
-            var expected = JObject.Parse(@"{
-              ""fields"": {
-                ""Boolean"": {
-                  ""boolean"": {
+            var expected = JObject.Parse($@"{{
+              ""indices"": [
+                ""{typesIndexFullName.Split(':')[1]}""
+              ],
+              ""fields"": {{
+                ""Boolean"": {{
+                  ""boolean"": {{
                     ""aggregatable"": true,
                     ""searchable"": true,
                     ""type"": ""boolean""
-                  }
-                },
-                ""DateTime"": {
-                  ""date"": {
+                  }}
+                }},
+                ""DateTime"": {{
+                  ""date"": {{
                     ""aggregatable"": true,
                     ""searchable"": true,
                     ""type"": ""date""
-                  }
-                },
-                ""Guid"": {
-                  ""string"": {
+                  }}
+                }},
+                ""Guid"": {{
```

Comment:
why the extra '{' on each line

Reply:
Needed to escape the braces characters in interpolated strings:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated